int32 rows
int32 cols
float32[] data
int32[] ids
float64[] time
float32 carrot_x
float32 carrot_y
float32 carrot_theta


